### DISCUSSION TITLE
Operations on Plaintext Vectors

### DISCUSSION THREAD

**Discussion by eneht-a** [openfheorg/openfhe-development#209] [Q&A]
Hi all,

As I couldn't find this in the sample code: 
Is it implemented to run additions/multiplications between ciphertexts which encrypt vectors of values (i.e., they are packed) and plaintext vectors? To be precise, I mean *not* as it is done in the CKKS example where a multiplication of a packed ciphertext with one fixed scalar is done. Instead, I would like to have a different factor/summand per element/slot in the ciphertext?

Many thanks!

**Comment by yspolyakov** [comment]
All discussions happen in the discourse forum: https://openfhe.discourse.group/ Please post your question there.
